src/libraries/System.Runtime/tests/System.Runtime.InteropServices.RuntimeInformation.Tests/DescriptionNameTests.cs

```diff
@@ -198,11 +198,21 @@ public void VerifyFrameworkDescriptionContainsCorrectVersion()
             Assert.DoesNotContain("+", version); // no git hash
 
 #if STABILIZE_PACKAGE_VERSION
-            // a stabilized version looks like 8.0.0
-            Assert.DoesNotContain("-", version);
-            Assert.True(Version.TryParse(version, out Version _));
+            // A stabilized version looks like 9.0.0
+            // In local builds, the version looks like 9.0.0-dev, and in PRs CI, the version looks like 9.0.0-ci.
+            // Both are acceptable.
+            if (!version.Contains("-dev") && !version.Contains("-ci"))
```

Comment:
This is not correct. The FrameworkDescription should always contain the stabilized version irrespective of the build environment.

This is a regression caused by https://github.com/dotnet/runtime/pull/102164. Before it was reading the AssemblyInformationalVersion from System.Private.CoreLib.dll which is `9.0.0`, now it's reading it from the generators assembly and that contains `9.0.0-dev` (basically what Stephen noticed in https://github.com/dotnet/runtime/pull/102164#discussion_r1601772016)

Reply:
I pushed a fix. There is potential to clean up overriding the version in SPC.dll now I think but that can be done separately.